test/s3.test.ts

```diff
@@ -0,0 +1,21 @@
+import { AWS } from "../src";
+
+const S3 = new AWS.S3();
+
+test("S3 PutObject and GetObject should work", async () => {
+  const Key = "test-key";
+  const Body = "test-body";
+
+  await S3.putObject({
+    Bucket: "itty-s3-bucket",
+    Key,
+    Body,
+  });
+
+  const response = await S3.getObject({
+    Bucket: "itty-s3-bucket",
+    Key,
+  });
+
+  expect(response?.Body?.toString()).toEqual(Body);
```

Comment:
I wonder how well this works with Streaming IO? We will have to test it with a big file - it's important to be able to stream io in and out rather than rely on the entire object always fitting in memory.